app/controllers/main_controller.rb

```diff
@@ -1,6 +1,37 @@
 class MainController < ApplicationController
                        
   def index
-    @card = Card.new
+    last_review_date = Date.today - 3.days
+    # last_review_date = 1.year.ago
+    start = 1.year.ago
+    @cards_for_learning = Card.where(review_date: start..last_review_date )
+    if @cards_for_learning.empty?
+      flash[:no_cards] = "Нет карточек, приходите через день или добавьте новых"
+    else
+    @card = @cards_for_learning.sample
+    end
   end
+
+  def check 
```

Comment:
Assignment Branch Condition size for check is too high. [19.05/15]<br>Trailing whitespace detected.